1-exercises/A-array-find/exercise.js

```diff
@@ -19,7 +19,10 @@ let names = [
 
 let longNameThatStartsWithA = findLongNameThatStartsWithA(names);
 
-console.log(longNameThatStartsWithA);
+
+
+const result = names.find((el) => el. startsWith("A") && el.length > 7)
```

Comment:
Nice, remove the let longNameThatStartsWithA = findLongNameThatStartsWithA(names); line and  it'll work properly